app/src/main/java/org/simple/clinic/newentry/PatientEntryModel.kt

```diff
@@ -58,4 +58,9 @@ data class PatientEntryModel(
 
   private fun getPersonalDetails(patientEntry: OngoingNewPatientEntry?): PersonalDetails =
       patientEntry?.personalDetails ?: PersonalDetails("", null, null, null)
+
+  fun withDistrict(district: String): PatientEntryModel {
+    val address = patientEntry?.address ?: Address("", "", "")
```

Comment:
Should we extract `patientEntry?.address ?: Address("", "", "")` into a separate function like we did for `PersonalDetails`? This is being used in three places.

Reply:
Yes, this I am refactoring this class. This will land in the next PR.